src/MarkerCluster.js

```diff
@@ -26,14 +26,16 @@ export var MarkerCluster = L.MarkerCluster = L.Marker.extend({
 	},
 
 	//Recursively retrieve all child markers of this cluster
-	getAllChildMarkers: function (storageArray) {
+	getAllChildMarkers: function (storageArray, ignoreDraggedMarker) {
```

Comment:
This will need to be included in the documentation.

Reply:
Yes, certainly. Should I  write a proposal?

Reply:
I have added a sentence to README.md, also indentation is now fixed